MobileCenterDistribute/MobileCenterDistribute/Internals/Model/MSReleaseDetails.m

```diff
@@ -134,7 +150,7 @@ - (NSDictionary *)serializeToDictionary {
   }
   if (self.uploadedAt) {
     NSDateFormatter *formatter = [NSDateFormatter new];
-    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ssZZZZZ"];
+    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"];
```

Comment:
I'm wondering why the date format was THIS weird in the first place.

Reply:
It was because the date string format was like `-0800`.